### PR TITLE
extensions: Small changes based on focss conversion work

### PR DESCRIPTION
**Pull request by Attamusc**

### CODE REVIEW

File: src/util/expression.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import jsep from 'jsep';
+      import jsep from 'jsep';
```
Comment (kaicataldo) [inline src/util/expression.js]: Extra whitespace

File: src/extensions/scale.js [hunk 2]
```diff
@@ -50,3 +60,12 @@ export function scaleLineHeight(fontSize, lineHeight, scalingFn = scaleText) {
 
   return scaledLineHeight;
 }
+
+export default api = {
```
Comment (kaicataldo) [inline src/extensions/scale.js]: Do you think we should rename these at all? `scaleApi.scale` isn't great. I think something like `scale.default()`, `scale.small()`, `scale.textSmall()`, etc. might be a nicer API

Comment (Attamusc) [inline src/extensions/scale.js]: This is only for the tests right now...I can change the way it's used to this is that makes more sense now

Comment (kaicataldo) [inline src/extensions/scale.js]: Cool, no worries then

**Pull request merged**